src/main/java/org/gridsuite/study/server/repository/LoadFlowResultEntity.java

```diff
@@ -1,53 +0,0 @@
-/**
- * Copyright (c) 2020, RTE (http://www.rte-france.com)
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- */
-
-package org.gridsuite.study.server.repository;
-
-import lombok.*;
-
-import javax.persistence.*;
-import java.util.*;
-
-/**
- * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
- * @author Etienne Homer <etienne.homer at rte-france.com>
- * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
- */
-@NoArgsConstructor
-@AllArgsConstructor
-@Getter
-@Setter
-@Entity
-@Table(name = "loadFlowResult")
-public class LoadFlowResultEntity {
-
-    public LoadFlowResultEntity(boolean ok, Map<String, String> metrics, String logs, List<ComponentResultEmbeddable> componentResults) {
-        this(null, ok, metrics, logs, componentResults);
-    }
-
-    @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
-    @Column(name = "id")
-    private UUID id;
-
-    @Column(name = "ok")
-    private boolean ok;
-
-    @Column(name = "metrics")
-    @ElementCollection
-    @CollectionTable(foreignKey = @ForeignKey(name = "loadFlowResultEntity_metrics_fk"))
```

Comment:
metrics are not stored in the new DB loadlow. It was not used ? We are sure that metrics are useless now ?

Reply:
This field is never used and it is always empty in the response from Powsybl

Reply:
Ok to remove it